packages/ag-charts-community/src/scene/sceneDebug.ts

```diff
@@ -48,6 +85,11 @@ export function debugStats(
     extraDebugStats = {},
     seriesRect = BBox.zero
 ) {
+    if (!Debug.check(DebugSelectors.SCENE_STATS, DebugSelectors.SCENE_FPS)) {
```

Comment:
This condition is the same as at the top of `debugFps()` so will it not always just return out of that too?